Intersect.Client.Framework/Entities/Label.cs

```diff
@@ -1,10 +1,9 @@
-﻿using Intersect.Client.Framework.Entities;
-using Newtonsoft.Json;
+﻿using Newtonsoft.Json;
 
-namespace Intersect.Client.Entities
+namespace Intersect.Client.Framework.Entities
 {
 
-    public struct Label : ILabel
+    public class Label
```

Comment:
I actually wouldn't have changed this away from a struct, but I guess this isn't really a memory issue and is fine as a class.